Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.cpp

```diff
@@ -1433,6 +1434,57 @@ void NetworkConnectionToWebProcess::installMockContentFilter(WebCore::MockConten
 }
 #endif
 
+#if ENABLE(LOGD_BLOCKING_IN_WEBCONTENT)
+void NetworkConnectionToWebProcess::logOnBehalfOfWebContent(const String& logChannel, const String& logString, uint8_t logType)
+{
+    WTFLogChannel* channel = getLogChannel(logChannel);
```

Comment:
Does network process here know about PAL, WebCore and WebKit log channel names?

Reply:
Good point! I have added these.

Thanks for reviewing :)